docs/GettingStarted/configuration/ExporterCommittime.md

```diff
@@ -1,9 +1,18 @@
 # Commit Time Exporter
 
 The job of the commit time exporter is to find and associate time of the relevant source code commit with a container image SHA built from that source code.
+
+This information can be found in two ways:
+
+- [git](#using-commit-time-with-git-apis) making a request to the Git provider
```

Comment:
Git API instead of just Git?

Reply:
Ok modified to:

[Git API]
[Containers' Image]